source/access.tex

```diff
@@ -116,8 +116,9 @@
 the class appearing outside the class's \grammarterm{member-specification}{.}
 \begin{note} This access also applies to implicit references to constructors,
 conversion functions, and destructors. \end{note}
-\begin{example}
 
+\pnum
+\begin{example}
```

Comment:
Consider here (and everywhere) whether the example illustrates the note or the section in general. (I'll take a look, too, but it's surprisingly hard to make GitHub show more context of the change.)